core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1679,6 +1681,32 @@ public void testCopyIn() throws SQLException {
         }
     }
 
```

Comment:
Formatting is a bit weird here .
Do you use default intellij formatter ?

Reply:
ha! well spotted! I didnt use IDE formatted, I tried to format it manually to keep it consistent with the rest of the code. it seems I failed :)